ax/models/torch/tests/test_sebo.py

```diff
@@ -317,19 +317,22 @@ def test_optimize_l0(self, mock_optimize_acqf_homotopy: Mock) -> None:
             fixed_features=self.fixed_features,
             options={"penalty": "L0_norm", "target_point": self.target_point},
         )
-        with self.assertRaisesRegex(
-            NotImplementedError,
-            "Homotopy does not support optimization with inequality "
-            + "constraints. Use L1 penalty norm instead.",
-        ):
-            acquisition.optimize(
-                n=2,
-                search_space_digest=self.search_space_digest,
-                inequality_constraints=self.inequality_constraints,
-                fixed_features=self.fixed_features,
-                rounding_func=self.rounding_func,
-                optimizer_options=self.optimizer_options,
-            )
+        acquisition.optimize(
+            n=2,
+            search_space_digest=self.search_space_digest,
+            inequality_constraints=self.inequality_constraints,
+            fixed_features=self.fixed_features,
+            rounding_func=self.rounding_func,
+            optimizer_options=self.optimizer_options,
+        )
+        args, kwargs = mock_optimize_acqf_homotopy.call_args
```

Comment:
The unused assignment may upset the linter otherwise. 

```suggestion
        kwargs = mock_optimize_acqf_homotopy.call_args.kwargs
```